### PR TITLE
docs(zone.js): update comment and chapter format of ngzone doc.

### PR DESCRIPTION
**Pull request by JiaLiPassion**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

Update some docs of ngZone document.
1. The chapter level is wrong.
2. Update the comment of the sample code.

### CODE REVIEW

File: aio/content/guide/zone.md [hunk 1]
```diff
@@ -335,12 +335,13 @@ In that situation, you can use another NgZone method: [runOutsideAngular()](api/
 export class AppComponent implements OnInit {
   constructor(private ngZone: NgZone) {}
   ngOnInit() {
-    // you know no data will be updated
-    // you don't want to do change detection in this
-    // specified operation, you can call runOutsideAngular
+    // You know no data will be updated
```
Comment (kapunahelewong) [inline aio/content/guide/zone.md]:
```suggestion
    // You know no data will be updated,
```

File: aio/content/guide/zone.md [hunk 2]
```diff
@@ -335,12 +335,13 @@ In that situation, you can use another NgZone method: [runOutsideAngular()](api/
...
+    // you don't want to trigger change detection in this
```
Comment (kapunahelewong) [inline aio/content/guide/zone.md]:
```suggestion
    // so you don't want to trigger change detection in this
```

File: aio/content/guide/zone.md [hunk 3]
```diff
@@ -335,12 +335,13 @@ In that situation, you can use another NgZone method: [runOutsideAngular()](api/
...
+    // specified operation, you can call ngZone.runOutsideAngular()
```
Comment (kapunahelewong) [inline aio/content/guide/zone.md]:
```suggestion
    // specified operation. Instead, call ngZone.runOutsideAngular().
```

File: aio/content/guide/zone.md [hunk 4]
```diff
@@ -335,12 +335,13 @@ In that situation, you can use another NgZone method: [runOutsideAngular()](api/
...
     this.ngZone.runOutsideAngular(() => {
       setTimeout(() => {
-        // do something will not update component data
+        // update data of the component
```
Comment (kapunahelewong) [inline aio/content/guide/zone.md]:
```suggestion
        // update component data
```

File: aio/content/guide/zone.md [hunk 5]
```diff
@@ -335,12 +335,13 @@ In that situation, you can use another NgZone method: [runOutsideAngular()](api/
...
+        // but will not trigger change detection.
```
Comment (kapunahelewong) [inline aio/content/guide/zone.md]:
```suggestion
        // but don't trigger change detection.
```

**Review by kapunahelewong — COMMENTED** [review]
Just some small suggestions. Thank you, @JiaLiPassion!

**Comment by JiaLiPassion** [comment]
@kapunahelewong , thank you for the fix, I have updated the PR.

**Review by kapunahelewong — APPROVED** [review]

**Pull request closed without merging**